docs/how-to-guides/client-supplied-encryption.md

```diff
@@ -0,0 +1,72 @@
+# Client Supplied Encryption
+
+PKI (Public Key Infrastructure) encryption technology is leveraged to provide additional security for customers using the VoiceBase platform. The customer provides us with their PGP "public key", which can be used to encrypt the data, but not to decrypt it. The customer keeps their "private key" and never sends it to VoiceBase. When the media is processed, we encrypt the results using the customer's "public key" for storage and delivery to the customer. The customer receives the encrypted data, and uses the "private key" to decrypt and use the results.
+
+Users may create a PGP key using PGP or GPG tools. We recommend to use an "RSA and RSA" key with 2048 bits, with one-year crypto-period.
+
+### Upload the public key to VoiceBase
+
+```bash
+curl -s "https://apis.voicebase.com/v3/security/public-keys"
+--header "Authorization: Bearer XXXXXX"
+--header "Content-Type: application/octet-stream"
+--data-binary @./jsmith-public-key.asc
+```
+
+The upload will return a "publicKeyId". Use this ID when uploading the media.
+At this point the key is ready to be used.
+
+### Using POST /v3/media with a public key
+
+Add the publicKeyId in your v3 API Configuration:
+
+```json
+{
+    ...
+    "encryption" {
+        "publicKeyId" : "the unique identifier"
+     },
+    "searchability" : {
+        "searchable" : false
+    }
+     ...
+}
+```
+
+#### Behavior of GETs, POSTs and DELETE when Client Supplied Encryption has been applied:
+
+```GET /v3/media?query={query}``` Because the media is encrypted, you will be unable to search by words in the transcript or metadata.
+
+```GET /v3/media/{mediaId}``` will display processing progress, but will not show the transcript contents.
+mediaQuery — will not index or show encrypted media.
```

Comment:
@Daniel085 I don't think our customers know these API by these names, let's use the HTTP methods and paths to be consistent with the rest of our documentation.